pyproject.toml

```diff
@@ -43,6 +43,7 @@ ascii_magic = "2.3.0"
 dictalchemy3 = "1.0.0"
 fire = "0.4.0"
 ipdb = "0.13.11"
+kivy = "2.2.1"
```

Comment:
Please remove if unused

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/290/commits/3a66d8830119123d73b290ce42af9aa370c3abb3